docs/references/experimental/vsi/README.md

```diff
@@ -0,0 +1,74 @@
+## Vendored Source Identification
+
+Some projects, especially C or C++ projects, vendor open source libraries in directories adjacent to the first party code being developed in-house.
+FOSSA's Vendored Source Identification is intended to support identifying and categorizing such vendored libraries.
+
+### How does it work?
+
+FOSSA fingerprints all files in your project and uploads those fingerprints to our analysis service.
+The analysis service then uses a proprietary algorithm to compare those fingerprints with the fingerprints in our database of open source projects.
+
+### Prerequisites
+
+VSI support requires a feature flag enabled in your FOSSA organization. If you're interested in using this feature please contact us!
+
+### FAQ
+
+#### Why not always enable VSI?
+
+VSI is more computationally intensive, and therefore takes longer to run.
+For this reason, we recommend only enabling it when you are reasonably confident that you will obtain useful information by enabling it.
+
+To explain, enabling VSI causes the CLI to:
+
+1. Fingerprint all files in the project
+2. Send those fingerprints to the FOSSA analysis service for vendored source identification
+3. Wait for remote analysis to complete
+
+This process has undergone a lot of optimization to improve its performance (and this is an area we’re continually investing in), but it will never be as fast as scanning local files for dependency information.
+
+#### Failed to resolve dependencies for FOSSA projects
+
+> _Note: This example mentions functionality provided by the "multi-stage builds" family of features; it's documented here because you're likely to see this error during an `--experimental-enable-vsi` run._
+> _For more information on multi-stage builds see the [multi stage builds overview](../msb/README.md)._
+
+Let's say we have two C projects: `LibProject` and `CliProject`.
+`LibProject` is a library that is used inside `CliProject`, and we want to report that relationship in FOSSA, such that dependencies of `LibProject` appear as deep dependencies of `CliProject`.
+
+Let's go further and say our CI pipeline for `LibProject` looks like this, where we compile its output binary (`libproject.o`) into the directory `out/`:
+
+```bash
+make build -o out
+```
+
+We can then integrate FOSSA here to essentially tie the fingerprint of that `libproject.o` binary to `LibProject` in FOSSA:
```

Comment:
[question] what happens if there are multiple binaries within out dir? do they all get associated with the `LibProject`?  Is this presuming we are scanning only Libproject right now? 

```
make build -o out
fossa analyze --experimental-link-project-binary out --experimental-enable-vsi --project LibProject??
```

Reply:
> what happens if there are multiple binaries within out dir? do they all get associated with the LibProject?

Correct.